1-introduction/introduction_Task1.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1, 101):
```

Comment:
There are no questions to the implementation, everything is good, but I would ask to formalize all tasks as a separate function with docstrings and description of variables, and the main part. Like following

```python
def function_name(var_name: var_type) -> function_tupe:
    """Function description.

    Vars description
    """
    pass


if __name__ == "__main__":
    function_name(var_name=var_value)
```

This is good code writing practice, so get used to writing like this right away. This comment also applies to all tasks, including future ones.

Reply:
Fixed. Thanks!